src/components/dialog/ObservationDialog.vue

```diff
@@ -14,7 +14,7 @@ Licensed under the Elastic License 2.0. */
     ObservationSchedule,
     StudyStatus,
     ValidationReport,
-  } from '../../generated-sources/openapi';
+  } from '@/gs/models';
```

Comment:
why is this an alias import?

Reply:
Let's more this discussion to one single source of conversation, please refer to https://github.com/MORE-Platform/more-studymanager-frontend/pull/585#discussion_r2053591102